src/ui/trigger_editor.ui

```diff
@@ -704,7 +704,7 @@
                  <item>
                   <widget class="QToolButton" name="button_toggleSearchAreaResults">
                    <property name="toolTip">
-                    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Toggles the display of the search results area.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
+                    <string>Toggles the display of the search results area.</string>
```

Comment:
The tool-tip parser produces _compact, auto-word wrapped_ tool-tips from HTML that takes up a minimum amount of on-screen space and that will fit in the space available and thus tends to avoid obscuring the underlying dialogue/widget.  Plain text tool-tips produce a single, possible very long, line of text that may not even fit on a small screen, also, HTML does allow the use of extra formatting for certain words or phrases that otherwise have \*\*crude\*\* approximations (see the button for the Central Debug Console) to produce emphasis.  It is my believe that **HTML rules** in tool-tips...